Tests/eppo/UniversalFlagConfigTests.swift

```diff
@@ -22,9 +22,6 @@ final class UniversalFlagConfigTest: XCTestCase {
     }
 
     func testDecodeUFCConfig() {
-        let config = try! UniversalFlagConfig.decodeFromJSON(from: UFCTestJSON)
-        XCTAssertEqual(config.flags.count, 14, "There should be 14 flags present in the JSON test file.")
```

Comment:
not a good assertion, it always changes

Reply:
😬 yup no reason to test for this either; if there is an issue parsing the flags some other tests will fail